core/src/test/scala/unit/kafka/log/remote/RemoteIndexCacheTest.scala

```diff
@@ -598,4 +615,11 @@ class RemoteIndexCacheTest {
       timeIndex.flush()
     }
   }
+
+  private def createCorruptRemoteIndexCacheOffsetFile(): Unit = {
+    val pw =  new PrintWriter((remoteOffsetIndexFile(new File(tpDir,RemoteIndexCache.DIR_NAME),rlsMetadata)))
```

Comment:
nit

spaces after `,` for each argument

Reply:
done

Reply:
You will notice that checkstyle is failing in the build (you can check the CI by clicking the "details" button below) since we still didn't add space after each comma.

```
* What went wrong:

Execution failed for task ':storage:checkstyleMain'.

> A failure occurred while executing org.gradle.api.plugins.quality.internal.CheckstyleAction

   > Checkstyle rule violations were found. See the report at: file:///home/jenkins/workspace/Kafka_kafka-pr_PR-14459/storage/build/reports/checkstyle/main.html

     Checkstyle files with violations: 1

     Checkstyle violations by severity: [error:1]
```

I would suggest to run `./gradlew unitTest` locally to ensure that build is successful before updating the PR.

Reply:
Yes @divijvaidya I will do it from next time , but is there a way to open this file , it seems to be created in build server ?
"file:///home/jenkins/workspace/Kafka_kafka-pr_PR-14459/storage/build/reports/checkstyle/main.html"

Reply:
No, unfortunately not. The best way to get the checkstyle report is to run it locally.

Reply:
@divijvaidya  fixed the build issue.

Reply:
@iit2009060 You can run the below command to catch different errors related to check-style, license header checks etc and also runs on different scala versions. It is good to run it before raising PR to catch errors. 

```
./gradlewAll clean releaseTarGz checkStyleMain checkStyleTest spotBugsMain spotBugsTest rat
```